resources/function-controller/templates/tests/test.yaml

```diff
@@ -16,12 +16,6 @@ spec:
       annotations:
         sidecar.istio.io/inject: "true"
     spec:
-      {{ if .Values.global.isLocalEnv }}
```

Comment:
This is not needed for minikube?

Reply:
Currently, we don't know how to handle minikube and it points to storage that is completely not used in this test... so I removed this to not confuse others.

Reply:
but it "works" if you add that specific lambda name to /etc/hosts

maybe we could keep it for posterity?

Reply:
hm... I will check as I'm waiting for rename merge ;)

Reply:
@aerfio ok, I'm still thinking that it doesn't make sense to keep it, the function has generated FQDN (like `test-function-1234566.kyma.local`), not static.